### PR TITLE
Copter: Disable throw mode

### PR DESCRIPTION
**Pull request by muramura**
There are no videos of the mode of throwing vehicles, except for ArduPilot members.
There are almost no scenes that use the vehicle throw mode.
I would disable this flight mode.

There is no video uploaded of a micro drone with ArduPilot hovering in throw mode.

### CODE REVIEW

**Comment by rishabsingh3003** [comment]
Ah, I really like the throw mode. Here is a video for you where I run the "drop" version of throw mode: https://youtube.com/shorts/uUK8zFFb0Z0?feature=share
It works really well!
But yeah, I haven't seen many people use it, so disabling by default seems okay to me

File: ArduCopter/config.h [hunk 1]
```diff
@@ -588,6 +588,7 @@
 # define AUTO_DISARMING_DELAY  10
 #endif
 
+#ifndef MODE_THROW_ENABLED
```
Comment (khancyr) [inline ArduCopter/config.h:591]: #if MODE_THROW_ENABLED

**Review by khancyr — CHANGES_REQUESTED** [review]
I don't thing there is much use of the the throw/drop mode. it is really usefull for small vehicle like skyviper but we should be able to handle the inclusion manually

File: ArduCopter/config.h (hunk 1 above)
Comment (muramura) [reply ArduCopter/config.h:591]: Thanks. I will change.

**Comment by rmackay9** [comment]
I've been quiet on this but I'm not sure I'm going to merge this.. It's true that Throw mode is not very widely used but it is a bit unique and fun...

**Comment by MichelleRos** [comment]
I've used throw mode. Is very fun/amusing, however I can see how it can't really be used on the majority of Copters people use ArduPilot for (given how most aren't really "throwable"). Mine's a 7 inch which is already small enough that various parts of AP struggles with it, yet it's unnerving enough to try throw mode on it that I wouldn't wanna try it with anything bigger...

I agree @rmackay9 that we shouldn't get rid of the mode totally, but given flash space limitations and the fact that this saves ~2.2k it looks like, perhaps we should think about disabling it on 1MB boards by default? (And allow people to use the custom build server if they want it.)

**Comment by peterbarker** [comment]
I'd think the 1MB copters are where this is most likely to be used.

I *do* use this mode on SkyViper and have mucked around with it on other vehicles.  This PR would obviously remove it from SkyViper.

I really don't think this is one to merge, myself.

**Comment by andyp1per** [comment]
Yes I agree, closing

**Comment by muramura** [comment]
Thank you for the review. I understand the reasons for the close.

**Pull request closed without merging**